### PR TITLE
Add Vercel sponsorship

### PR DESCRIPTION
**Pull request by sanny-io**

### CODE REVIEW

**Comment by sanny-io** [comment]
@dishwasher-detergent check the Vercel preview link and let me know if the logo placement is ok. It appears in the home page hero container and the article sidebar.

File: package.json [hunk 1]
```diff
@@ -21,11 +21,12 @@
     "algoliasearch": "^4.14.2",
     "dotenv": "^16.0.3",
     "framer-motion": "^6.5.1",
+    "glob": "^9.3.4",
     "glob-promise": "^6.0.0",
     "gray-matter": "^4.0.3",
     "lucide-react": "^0.92.0",
     "next": "^13.1.5",
-    "next-mdx-remote": "^4.2.0",
+    "next-mdx-remote": "^4.4.1",
```
Comment (dishwasher-detergent) [inline package.json]: Were you able to get it working with 4.4.1? I kept having an issue, I reverted to 4.2.0 after reading others having the same issue.

File: package.json [hunk 2]
```diff
@@ -21,11 +21,12 @@
     "algoliasearch": "^4.14.2",
     "dotenv": "^16.0.3",
     "framer-motion": "^6.5.1",
+    "glob": "^9.3.4",
     "glob-promise": "^6.0.0",
```
Comment (dishwasher-detergent) [inline package.json]: I don't think glob promise was actually being used. You can delete it.

**Review by dishwasher-detergent — CHANGES_REQUESTED** [review]
Remove glob-promise and it's import from pages/api/article/index.tsx and it should be good . I was getting conflicts with glob-promise and glob. But I don't see anywhere glob promise is still being used.

File: package.json (hunk 1 above)
Comment (sanny-io) [reply package.json]: Woops, that was not intended to be pushed. Was experimenting with the infinite load I was having. Reverted this.

**Review by dishwasher-detergent — APPROVED** [review]

**Pull request merged**